Gordon360/Services/ProfileService.cs

```diff
@@ -359,6 +373,68 @@ public async Task UpdateImagePrivacyAsync(string username, string value)
 
             return profile.ToObject<ProfileViewModel>();
         }
+        public async Task InformationChangeRequest(string username, ProfileFieldViewModel[] updatedFields)
+        {
+            var account = _context.ACCOUNT.FirstOrDefault(x => x.AD_Username == username);
+            if (account == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The account was not found." };
+            }
```

Comment:
The preferred way to do this is by using the AccountService, which will handle the database access and not found behavior for you:
```suggestion
            var account = _accountService.GetAccountByUsername(username);
```